install-dev/upgrade/sql/1.7.7.0.sql

```diff
@@ -5,6 +5,7 @@ ALTER TABLE `PREFIX_search_word` MODIFY `word` VARCHAR(30) NOT NULL;
 INSERT INTO `PREFIX_configuration` (`name`, `value`, `date_add`, `date_upd`) VALUES ('PS_DISPLAY_MANUFACTURERS', '1', NOW(), NOW());
 
 /* PHP:ps_1770_preset_tab_enabled(); */;
+/* PHP:ps_1770_update_order_status_colors(); */;
```

Comment:
I saw methods are "registered" this way, I assume they are executed somehow? :thinking:

Reply:
yes, with this code you call the script `ps_1770_update_order_status_colors.php` and the function with the same name :)

Reply:
I'm confused, I don't know if that's cool or just a hack. :boom: